app/src/main/java/com/byoutline/kickmaterial/features/projectdetails/ProjectDetailsActivity.kt

```diff
@@ -113,9 +114,9 @@ class ProjectDetailsActivity : KickMaterialBaseActivity(), DelayedTransitionActi
     }
 
     private fun showRewardList() {
-        transitionHelper.executeIfCachedProjectDetailsAreAvailable(R.string.retry_getting_project_rewards) { details ->
+        transitionHelper.executeIfCachedProjectDetailsAreAvailable({ details ->
             RewardsListActivity.launch(this, details, binding.playVideoBtn)
-        }
+        }, displayErrorAction = { showToast(R.string.retry_getting_project_details) })
```

Comment:
Not strictly related. Caused by SecretSauce update (where it is easier to show toasts from Fragment/Activity then from viewModel). It is probably more view related anyway.